esp32/modesp.c

```diff
@@ -101,6 +146,17 @@ STATIC const mp_rom_map_elem_t esp_module_globals_table[] = {
 
     { MP_ROM_QSTR(MP_QSTR_neopixel_write), MP_ROM_PTR(&esp_neopixel_write_obj) },
     { MP_ROM_QSTR(MP_QSTR_dht_readinto), MP_ROM_PTR(&dht_readinto_obj) },
+
+#if MODESP_INCLUDE_CONSTANTS
+    { MP_OBJ_NEW_QSTR(MP_QSTR_LOG_NONE), MP_OBJ_NEW_SMALL_INT((mp_uint_t)ESP_LOG_NONE)},
+    { MP_OBJ_NEW_QSTR(MP_QSTR_LOG_CRITICAL), MP_OBJ_NEW_SMALL_INT((mp_uint_t)ESP_LOG_ERROR)},
```

Comment:
The levels are defined as an enum and typedef in esp-idf/components/log/include/esp_log.h, and there isn't a ESP_LOG_CRITICAL. I wanted to provide some equivalence for the Python abstraction, though: the absence of the level in the base OS, shouldn't be something that your casual developer needs to worry about.